source/reference/api/commit.txt

```diff
@@ -15,7 +15,39 @@
 Description
 -----------
 
-Commits the synchronization operation to the destination cluster.
+Finalizes the sync between the source cluster and the destination
+cluster and stops continuous sync between clusters. ``commit`` also
+restores collection characteristics that ``mongosync`` temporarily
+alters during synchronization. 
+
+The following table illustrates the impact of ``commit`` on collection
+characteristics that ``mongosync`` alters during the synchronization process.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Collection Characteristic
+     - Impact of ``commit`` 
+   * - :ref:`Unique Indexes <c2c-dr-unique>`
+     - ``commit`` resets unique indexes that ``mongosync`` replicated as
+       non-unique on the destination cluster.
+
+   * - :ref:`TTL Indexes <c2c-dr-ttl>`
+     - ``commit`` resets ``expireAfterSeconds`` which ``mongosync`` set
+       to the value of ``MAX_INT`` on the destination cluster.
+
+   * - :ref:`Hidden Indexes <c2c-dr-hidden>`
+     - ``commit`` resets hidden indexes that ``mongosync``
+       replicated as non-hidden on the destination cluster.
+
+   * - :ref:`Write Blocking <c2c-dr-write-blocking>`
+     - If ``mongosync`` disabled writes on the destination cluster,
+       ``commit`` disables the Write Block Mode. 
+
+   * - :ref:`Capped Collections <c2c-dr-capped-collections>`
+     - ``commit`` resets the required maximum size of capped collections
+       which ``mongosync`` set to the maximum allowable size on the
+       destination cluster.
```

Comment:
Does this header duplicate the information in the disaster-recovery page?

If so, that seems like additional overhead and opportunity for confusion/inaccuracy, if those 2 pages now have to be kept manually in sync.

Is it possible to write the table out to a file, then include it separately? Maybe using rST’s [include](https://docutils.sourceforge.io/docs/ref/rst/directives.html#including-an-external-document-fragment) directive?

Reply:
The information in the commit table and the [disaster recovery table ](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/disaster-recovery/) is similar but has different use cases which is why I hesitate to make make them identical with an include file. For the disaster recovery page, the user needs to know which commands to use (for example, use the collMod command to set expireAfterSeconds to the desired value) to manually take action while the commit table does not need to provide that information. Because the disaster recovery table needs to include instructions on how to manually reset the changes as opposed to the commit table which is just describes how things happen automatically,  I would be in favor of leaving the two tables separate.

Reply:
That makes sense. Thanks!